pkg/prog/progtest/progtest.go

```diff
@@ -34,7 +56,8 @@ func (f *Fixture) Cleanup() {
 	f.dirCleanup()
 }
 
-// Fds returns the file descriptors in the fixture.
+// Fds returns the file descriptors in the fixture suitable for use by the
+// shell, not the test framework.
```

Comment:
What's exact meaning of "not the test framework" here - what kind of misuse do you have in mind?

Reply:
I was just being pedantic that the return value of this method is meant as a suitable (stdin, stdout, stderr) triplet for `shell.Script` and related functions. When I embarked on this work (really the fix for #661) I was initially confused by this method. Since I'm no longer confused I'll just revert this part of the change.